### DISCUSSION TITLE
Manga/Comic Readers

### DISCUSSION THREAD

**Discussion by OmiChrome** [metafates/mangal#88] [General]
i would like to give some of the best reaader i use to read manga-comics 🤩 

for linux : 
[zathura](https://pwmt.org/projects/zathura/)

for window  😉 : 
[html-mangareader](https://github.com/luejerry/html-mangareader)

👆 this are the best reader i have used till now . html-mangareader is charm for me 🌟

**Comment by OmiChrome** [comment]
@metafates  hey , i would like if u add this reader in readme.md file for easier acces to everyone :)

**Reply by metafates** [reply]
I think it's fine to have it as a discussion instead, so everyone could recommend their own app

**Comment by metafates** [comment]
Cool! 

I would recommend [yacreader](https://www.yacreader.com). It supports PDF & CBZ and has some [neat features](https://www.yacreader.com/features)

Also, [Komga](https://komga.org) is just great. Very convenient

**Reply by OmiChrome** [reply]
noice
